### ISSUE TITLE
Olin Uris Accessibility Fixes

### ISSUE THREAD

**Issue by sarah-cul** [cul-it/uls#1211]
siteimprove score is currently at 91.  To improve this score:

- [x] Links missing text alternative:  28 Links  on 10 pages. Easier to see these from siteimprove than to list here. Note, these seem to be for the gallery images in the elementor plugin, maybe there is a way to add the alt tags when declaring the images for this gallery in WPAdmin. example page: https://olinuris.library.cornell.edu/past-exhibitions/parallax/serviss/sfpioneer
- [x] Empty Heading: There is an empty anchor for maintenance inside a element, can be deleted. This is on the Graduate Locker Policies and Procedures page. https://olinuris.library.cornell.edu/olin-graduate-locker-policies-and-procedures
- [x] Insufficient Color Contrast on the "Clear Filter(s)" Button, found at bottom of the News & Events -> Exhibitions page. The entire Filter by: section doesn't seem to be useful here, maybe it can be removed entirely. Update: confirmed with content owner, it can be removed.

Found with ANDI:
- [x] 2 Elements with No Accessible Name (main page, Instagram and Blog icons under Latest Blog Posts)
- [ CAUSED BY DISTRIBUTOR PLUGIN ] 4 Ambiguous Link: same name/description as another link but different href. (Department menu, Department under filter facet plugin. Orcid logo link for 2 staff members)
- [x] Ambiguous Link under Services -> Instruction and Outreach, the workshop link simply has an extra trailing slash
- [x] Use the Library -> Spaces.  Interfaith Serenity and Meditation Room.  2 problems here: 
          1. Elements with No Accessible Name.  The image needs a name
          2. Below that, the button 'Located on Olin's basement level button'  links nowhere, try adding a role="none" or "note" to this. 
- [x] News & Events -> Exhibitions page.  at the bottom the Category and Tag links [<a> element has no [href], [id], or [tabindex]
- [ CAUSED BY LOCALIST WIDGET - they know of these issues ] News & Events -> Upcoming Events. 
           1.  The Choose Type selection under calendar has no accessible name [Select has no accessible name, associated <label>, or [title].]
           2. [4 Ambiguous Link: same name/description as another link but different href.]
                "Uris Library" links, should have title to distinguish them. 
                Cornell logo is also a link used multiple times
- [ CAUSED BY LOCALIST WIDGET ] News & Events -> Workshops
      The Choose Type selection under calendar has no accessible name [Select has no accessible name, associated <label>, or [title].]

- [ CAUSED BY ELEMENTOR ] Collapsable lists have issues. 
   -  About- > Visitor Information. Directions & Parking etc.
   -  About -> Frequently Asked Questions, the Borrowing & Returning Books etc. 
   -   Use the Library -> Laptop & Equipment Loans
   -   Resources -> Collection Highlights
